BlackMarket/app/SignUp/SignUpCardView.swift

```diff
@@ -0,0 +1,211 @@
+      LocalizedString.SignUpCardView.dataPolicy.transformToLink(
+        withURL: Constants.Path.dataPolicy
+      ) ?? ""
+    }
+    var cookiesPolicyLink: AttributedString {
+      LocalizedString.SignUpCardView.cookiesPolicy.transformToLink(
+        withURL: Constants.Path.cookiesPolicy
+      ) ?? ""
+    }
+  }
+  
+  enum Action {
+    case nameChanged(BMTextFieldDomain.Action)
+    case emailChanged(BMTextFieldDomain.Action)
+    case passwordChanged(BMTextFieldDomain.Action)
+    case registerUser
+    case showPolicy
+    case presentLogIn
+    case showCookiesPolicy
+  }
+  
+  var body: some ReducerProtocol<State, Action> {
+    Reduce { state, action in
+      switch action {
+      case .registerUser:
+        return .none
+      case .showPolicy:
+        return .none
+      case .presentLogIn:
+        return .none
+      case .showCookiesPolicy:
+        return .none
+      default:
+        return .none
+      }
+    }
+
+    Scope(state: \.nameState, action: /Action.nameChanged) {
+      BMTextFieldDomain()
+    }
+    Scope(state: \.emailState, action: /Action.emailChanged) {
+      BMTextFieldDomain()
+    }
+    Scope(state: \.passwordState, action: /Action.passwordChanged) {
+      BMTextFieldDomain()
+    }
+  }
+}
+
+struct SignUpCardView: View {
+  
+  let store: StoreOf<SignUpCardDomain>
+  
+  var body: some View {
+    WithViewStore(store, observe: { $0 }) { viewStore in
+      Spacer()
+      ZStack {
+        RoundedRectangle(cornerRadius: UI.CornerRadius.medium)
+          .foregroundColor(.white)
+        VStack {
+          Image.logo
+            .padding()
+          VStack(spacing: UI.Padding.verySmall) {
+            BMTextFieldView(
+              store: store.scope(
+                state: \.emailState,
+                action: SignUpCardDomain.Action.emailChanged
+              )
+            )
+            BMTextFieldView(
+              store: store.scope(
+                state: \.nameState,
+                action: SignUpCardDomain.Action.nameChanged
+              )
+            )
+            BMTextFieldView(
+              store: store.scope(
+                state: \.passwordState,
+                action: SignUpCardDomain.Action.passwordChanged
+              )
+            )
+            Button(action: {
+              //
+            }, label: {
+              BMButton(
+                backgroundColor: .black,
+                textColor: .white,
+                title: LocalizedString.SignUpCardView.signupButtonTitle,
+                cornerRadius: UI.CornerRadius.small
+              ).withStyle(.primary)
+            })
+            .padding(.top)
+            Spacer()
+            HStack {
+              Text(LocalizedString.SignUpCardView.policyDisclaimerFirstPart) +
+              Text("\(.init(viewStore.dataPolicyLink))") +
+              Text(LocalizedString.SignUpCardView.policyDisclaimerSecondPart) +
+              Text("\(.init(viewStore.cookiesPolicyLink))")
```

Comment:
pls check my comment about parameterized localized strings.

Reply:
Fixed in [this PR](https://github.com/tarruk/blackMarket/pull/7)